src/screens/Landing/LandingScreen.tsx

```diff
@@ -9,15 +9,15 @@ export const LandingScreen: React.FC<any> = ({navigation}) => {
     <View style={styles.blockMainScreen}>
       <Text style={styles.header}>Добро пожаловать в приложение</Text>
       <View style={styles.buttonEnter}>
-        <AppButton
+        <CustomButton
           title={'Вход'}
           onPress={() => {
             navigation.navigate(screenNames.LOGIN_SCREEN);
           }}
         />
       </View>
       <View style={styles.buttonRegistration}>
-        <AppButton
+        <CustomButton
           title={'Регистрация'}
           onPress={() => {
             navigation.navigate(screenNames.REGISTRATION_SCREEN);
```

Comment:
There should be NO arrow functions in JSX and in render() methods. Button onPress handlers should be declared separately. Simple handlers like navigation.navigate must be declared like default arrow functions. Complicated handlers should be wrapped in useCallback hook to optimize performance.